akka-typed/src/main/scala/akka/typed/ActorContext.scala

```diff
@@ -124,7 +124,7 @@ trait ActorContext[T] {
    * protocols can be ingested by this Actor. You are strongly advised to cache
    * these ActorRefs or to stop them when no longer needed.
    */
-  def spawnAdapter[U](f: U ⇒ T): ActorRef[U]
+  def spawnAdapter[U](f: U ⇒ T, name: String = ""): ActorRef[U]
```

Comment:
👍 
though i wonder if it would be safer to add them as overloads, in case we need to evolve these apis hmmm